main_test.go

```diff
@@ -25,6 +25,21 @@ func TestParsingBindStatement(t *testing.T) {
 	assert.Equal(t, pgLog.Database, "walle_test", "they should be equal")
 }
 
+func TestParsingShardPartition(t *testing.T) {
+	log := `2021-01-11 15:25:36 EST [56193-3/9939-5708] postgres@baller_test LOG:  duration: 0.020 ms  execute <unnamed>: SELECT * FROM abacus101_shard6.transactions WHERE balance = 13.37`
```

Comment:
Can we get an additional test for something like:

```
SELECT __user.id, __user.guid FROM "yolos_qa"."users" __user WHERE (__user.is_deleted = $1 OR __user.is_deleted is null) AND __user.guid IN ($2) AND __user.user_guid IN ($3) ORDER BY __user.id ASC LIMIT 1
```

I plucked that from QA logs and changed the service / model name. Does it pull the partition name correctly?

Reply:
ok, will do